src/main/java/com/google/api/codegen/transformer/php/PhpSurfaceNamer.java

```diff
@@ -178,4 +179,28 @@ private static String getTestPackageName(String packageName) {
   public boolean methodHasRetrySettings(MethodConfig methodConfig) {
     return !methodConfig.isGrpcStreaming();
   }
+
+  public String getRequestTokenFieldName(PageStreamingConfig pageStreaming) {
+    // Not using keyName since PHP maps the string "requestPageTokenField" to the raw request token
+    // field name from the page streaming config.
+    return pageStreaming.getRequestTokenField().getSimpleName();
```

Comment:
Yes. Right now fields in proto messages are lower underscore (https://github.com/googleapis/proto-client-php/blob/master/src/logging/v2/logging.php#L537) while keyName formats as lower camel (https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/util/php/PhpNameFormatter.java#L86).

Note also that it is probably worth doing only the minimum effort for PHP right now, because the proto implementation is going to change, and this will no longer be true.